File: app/src/main/java/arraymethod/ex01.java
package arraymethod;

import java.util.Arrays;

public class ex01 {
    public static void main(String[] args) {
        int[] scores = {90, 85, 95};

        System.out.println("array object: " + scores);
        System.out.println("array values: " + Arrays.toString(scores));

        for (int score : scores) {
            System.out.print(score + " ");
        }
        System.out.println();
    }
}

File: app-kotlin/src/main/kotlin/arraymethod/ex01.kt
package arraymethod

import java.util.Arrays

class ex01 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val scores = intArrayOf(90, 85, 95)
            println("array object: $scores")
            println("array values: ${Arrays.toString(scores)}")
            println(scores.joinToString(" "))
        }
    }
}
